python_backend/api/v1/endpoints/health.py

```
@@ -1,32 +1,87 @@
+            "rag_chains": True,
+            "finetune_service": True
+        }
+    )
+
+@router.get("/api/health",
+            response_model=HealthResponse, 
+            summary="API 헬스체크",
+            description="API 전용 헬스체크 엔드포인트입니다.")
+async def api_health_check() -> HealthResponse:
+    """API 전용 헬스체크"""
+    from datetime import datetime
```

Ruff (0.12.2)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)